src/Main/java/Game/Manager.java

```diff
@@ -33,12 +30,11 @@ public void AddActor(Actor a){
      */
     private static Manager INSTANCE;
     public static void Reset(){
-        timeInWater = new HashMap<Player, Integer>();
-        timeTent = new HashMap<Field,Integer>();
-        parts = new ArrayList<Item>();
```

Comment:
Ehelyett mi lett?

Reply:
Csak módosítva lettek.
A biztonság kedvéért lecsekkoltam a kommitot, a következő három sorban látszódik, mi került a helyükre.
Mivel az explicit típuskiírást el lehet hagyni, így azzal egyszerűsítettem, de máshogy nem bántottam őket.

Reply:
a parts-ra gondoltam, az komplett eltűnt

Reply:
Igazad van, a parts (és a currentPlayer is) ténylegesen el lett távolítva.
Ennek oka, hogy sehol nem használtuk őket.

Ebben a [commitban](https://github.com/BME-MIT-IET/iet-hf2021-iglue/pull/11/commits/529aaa8405533e0f75835dd906e28e29c9fc551d) láthatod pontosabban mi történt.
A commit message is erre utal:
> Remove unnecessary variables from Manager